app/src/main/res/layout/activity_main.xml

```diff
@@ -8,6 +8,7 @@
 
 
     <TextView
+        style="@style/MainText"
         android:id="@+id/main_text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
```

Comment:
Github won't let me comment on the specific line because you didn't change it this time, but it would probably be nice to extract the TrainBoard title into your strings.xml file